pcap-int.h

```diff
@@ -107,6 +107,107 @@ extern "C" {
  */
 #define MAXIMUM_SNAPLEN		262144
 
+/*
```

Comment:
OK, is there any other issues about this PR? I mean I can fix them all together:) So the next PR can be merged with one try.